web/docs/deployment/deployment-methods/wasp-deploy/ci-cd.md

```diff
@@ -0,0 +1,115 @@
+
+The workflow needs to include the following steps:
+
+1. Checkout the code from the repository
+2. Install Node.js and the Wasp CLI
+3. Install any provider-specific dependencies
+4. Deploy the application using `wasp deploy <provider> deploy`
+
+You need to make sure that the **provider-specific environment variables** (for example, provider API keys) are set up correctly before deploying. We show which variables are required and how to set them up in the Github Actions [example](#github-actions-workflow).
+
+
+## Github Actions workflow
+
+Let's take a look at an example CI/CD workflow for Github Actions for each of the supported providers.
+
+<Tabs>
+<TabItem value="fly" label="Fly.io">
+
+You'll need an **organisation token** to deploy to Fly.io. You can generate the token by running [`fly tokens create org`](https://fly.io/docs/security/tokens/#create-org-scoped-tokens) and adding it to your repository secrets as `FLY_API_TOKEN`.
+
+```yaml title=".github/workflows/deploy.yml"
+name: Wasp Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      # You must add FLY_API_TOKEN to your Repository Secrets
+      FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v6
+        with:
+          node-version: '22'
+
+      - name: Install Wasp
+        run: curl -sSL https://get.wasp.sh/installer.sh | sh -s -- -v 0.19.0
+
+      - name: Install Flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+
+      - name: Deploy
+        run: wasp deploy fly deploy
+```
+
+</TabItem>
+<TabItem value="railway" label="Railway">
+
+You'll need an **account token** to deploy to Railway. You can generate it by going to your account settings under [Tokens](https://railway.com/account/tokens) and generating a token without selecting a workspace. Set the token as a repository secret named `RAILWAY_API_TOKEN`.
+
+Make sure to replace `my-project-name` with the actual name of your project and `MY_PROJECT_ID` with the actual ID of your project. You can find the project ID in the project's settings.
+
+```yaml title=".github/workflows/deploy.yml"
+name: Wasp Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      # You must add RAILWAY_API_TOKEN to your Repository Secrets
+      RAILWAY_API_TOKEN: ${{ secrets.RAILWAY_API_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v6
+        with:
+          node-version: '22'
+
+      - name: Install Wasp
+        run: curl -sSL https://get.wasp.sh/installer.sh | sh -s -- -v 0.19.0
+
+      - name: Install Railway CLI
+        run: npm install -g @railway/cli
+
+      - name: Deploy
+        run: wasp deploy railway deploy my-project-name --existing-project-id MY_PROJECT_ID
```

Comment:
Maybe we take the PROJECT_ID parameter to a env here so that we can point user's attention to it better?